MetaMask/metamask-extension
# feat: add Numbers network and token icons

## **Description**

Add Numbers Protocol mainnet network icon and token icon support.

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Go to [Numbers Mainnet Explorer](https://mainnet.num.network/)
2. Scroll down to bottom and click the Metamask icon to add network
3. Open Metamask extension to check the network and token icons

## **Screenshots/Recordings**



### **Before**

<img width="357" alt="截圖 2024-05-14 下午1 22 26" src="https://github.com/MetaMask/metamask-extension/assets/24616552/1a95f3eb-7acb-477b-95a0-c010ee408652">


### **After**

<img width="355" alt="截圖 2024-05-09 下午4 46 02" src="https://github.com/MetaMask/metamask-extension/assets/24616552/ee764f57-3080-4733-b3b7-4c35b467f5ef">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
